core/client/fs/src/main/java/alluxio/client/file/cache/evictor/CacheEvictorOptions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.evictor;
+
+/**
+ * Options for initiating cache evictor.
+ */
+public class CacheEvictorOptions {
+
+  private boolean mIsNondeterministic;
+  private Class<? extends CacheEvictor> mEvictorClass;
+
+  /**
+   * @return if true, the evictor picks uniformly from the worst k elements
+   */
+  public boolean isNondeterministic() {
```

Comment:
Why need a separate member value? can this information be inferred from the CacheEvictor  class?

Reply:
i would expect that's a information that the evictorClass contains?

Reply:
this one is a configuration property regarding which evictor to load

Reply:
Interestig
  static CacheEvictor create(CacheEvictorOptions options) {
    if (options.isNondeterministic() && options.getEvictorClass().equals(LRUCacheEvictor.class)) {
      return new NondeterministicLRUCacheEvictor();
    }
i thought the evictor class will be directly pass in NondeterministicLRUCacheEvictor.class and contains this information, but the current way may be more user-friendly